Mobile/starter_project_matador/lib/features/example/presentation/screen/signup.dart

```diff
@@ -0,0 +1,216 @@
+// Import flutter heplper library
+import 'package:flutter/material.dart';
+
+class SignUp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+        title: 'Sign Up Page',
+        home: Scaffold(
+            backgroundColor: Colors.white,
+            body: SafeArea(
+                child: ListView(children: <Widget>[
+              SizedBox(
+                height: 130,
+                width: 234,
+                child: Image(
+                  height: 54,
+                  width: 141,
+                  image: NetworkImage(
```

Comment:
Declare a variable and assign it here instead of the full URL being fed to the widget. This makes integration easier.